File: cli/src/modules/dockerfile/tmux.ts
import type { DockerfileModule } from '@/core/types.js';

export const tmuxModule: DockerfileModule = {
  id: 'tmux',
  label: 'Tmux',
  category: 'infra',
  render() {
    return `##
## TMUX
##
RUN apt-get update && apt-get install -y tmux
`;
  },
};

File: cli/internal/modules/dockerfile/tmux.go
package dockerfile

import "github.com/joacohbc/my-devcontainer-installer/cli/internal/core"

var TmuxModule = &DockerfileModuleSpec{
	ID:       "tmux",
	Label:    "Tmux",
	Category: core.CategoryInfra,
	Render: func(opts map[string]any) string {
		return `##
## TMUX
##
RUN apt-get update && apt-get install -y tmux
`
	},
}
